jersey/src/main/java/com/netflix/conductor/server/resources/WorkflowBulkResource.java

```diff
@@ -55,67 +56,130 @@ public WorkflowBulkResource(WorkflowExecutor executor) {
         this.executor = executor;
     }
 
+    /**
+     * Pause the list of workflows.
+     * @param workflowIds
+     * @return bulk response object containing a list of succeeded workflows and a list of failed ones with errors
+     * @throws Exception
```

Comment:
What kind of `exception` is thrown ?  `IllegalArgument` exception dues to precondition check, java doc needs to specify that.